### PR TITLE
Ricardo rdz g cleaner install

### PR DESCRIPTION
**Pull request by RicardoRdzG**
Fixed several bugs in install script and added code for cleaner install

### CODE REVIEW

**Comment by RicardoRdzG** [comment]
@imachug Can you take a look at this PR as well as the one to integrate Travis for CI from @DemonixCraft ?

File: install.sh [hunk 1]
```diff
@@ -1,5 +1,37 @@
...
-mkdir -pv ~/bin/win-sudo && cd "$_";init-gitrepo-setup && gramf https://github.com/imachug/win-sudo.git && git pull -q base master
-echo "source ~/bin/win-sudo/s/path.sh" | tee -a ws-files null && echo "source ~/bin/win-sudo/s/aliases" >>~/.bashrc && ws-success
+function ws-script-start {
+  echo "Downloading win-sudo..."
+  trap 'echo "Failed to install, sorry :("; exit 1' ERR
+}
+function ws-success {
+  echo Win-sudo successfully installed!
+  exec bash
+  alias fresh="exec bash"
```
Comment (purplesyringa) [inline install.sh]: Shouldn't these 2 lines be swapped?

Comment (RicardoRdzG) [reply install.sh]: probably, I just moved those over from the other file I removed

**Comment by purplesyringa** [comment]
Thank you!

**Pull request merged**